src/extensions/utils.ts

```diff
@@ -55,3 +55,13 @@ export function getRandomString(length: number): string {
   }
   return result;
 }
+
+/**
+ * Formats a timestamp from the Extension backend into something more readable
+ * @param timestamp with this format: 2020-05-11T03:45:13.583677Z
+ * returns a timestamp with this format: 2020-05-11 T03:45:13
+ */
+export function formatTimestamp(timestamp: string): string {
+  const withoutMs = timestamp.split(".")[0];
+  return withoutMs.replace("T", " T");
```

Comment:
Probably just a space? I think `2020-05-11 03:45:13` is perfectly readable without the T which feels a bit odd.

Reply:
+1 to this